server/routes/user.js

```diff
@@ -0,0 +1,93 @@
+const express = require("express");
+const router = express.Router();
+const verifyAuth = require('../middleware/verifyAuth');
+const User = require("../models/user");
+const University = require("../models/universities");
+const validateBody = require('../middleware/validateBody');
+const validateCourseId = validateBody.course;
+require("../models/courses");
+
+
+// Get the logged in user
+router.get("/", verifyAuth, async function(req, res) {
+    const userId = req.body.userId;
+    const userDoc = await User.findById(userId)
+      .catch(() => { return null });
+    return userDoc ? res.send(userDoc) : res.sendStatus(500);
+})
+
+// Gets all the current users courses
+router.get("/course", verifyAuth, async function(req, res) {
```

Comment:
rename to 'courses' since we're getting all courses

Reply:
Okay! I was a little confused about this, because I read the route naming article you sent and I got the impression that if any of the routes contain a word that's singular then the whole route should be singular. 
For example the full route name of this is /user/course.

Reply:
Yeah, the convention would generally be to name the route after the collection it's interacting with, so to pull an example directly from the article:

/users/<userID>/courses/<courseID>

Many users -> specific user ID -> many courses for a user -> specific course ID